specificatie/openapi.yaml

```diff
@@ -558,6 +600,127 @@ components:
         pattern: '^[0-9]{16}$'
         example: '0226010000038820'
 
+    woonplaatsIdentificatie-query:
+      description: "De identificatie van een woonplaats."
+      name: woonplaatsIdentificatie
+      in: query
+      required: false
+      schema:
+        type: string
+        pattern: '^[0-9]{3}[1-9]|[0-9]{2}[1-9][0-9]|[0-9][1-9][0-9]{2}|[1-9][0-9]{3}$'
+        example: '2096'
+
+    geconstateerd:
+      description: 'Geef aan of je naar een geconstateerd object wilt zoeken.'
+      name: geconstateerd
+      in: query
+      required: false
+      schema:
+        type: boolean
+        default: false
+        
+    statusPand:
+      description: "Status van het pand."
+      name: statusPand
+      in: query
+      required: false
+      schema:
+        type: array
+        minItems: 1
+        items:
+          $ref: '#/components/schemas/StatusPandEnum'
+
+    bouwjaar:
+      description: >-
+        Bouwjaar van het pand.
+        Geef het minimale en/of maximale bouwjaar van het pand op waarnaar je wilt zoeken. 
+        Bv: bouwjaar[min]=1970&bouwjaar[max]=2010
+      name: bouwjaar
+      in: query
+      required: false
+      schema:
+        $ref: '#/components/schemas/IntegerFilter'
+      explode: true
+      style: deepObject
+
+    type:
+      description: "Geeft aan of het adresseerbaar object een ligplaats, standplaats of verblijfsobject is."
+      name: type
+      in: query
+      required: false
+      schema:
+        $ref: '#/components/schemas/TypeAdresseerbaarObjectEnum'
+
+    gebruiksdoelen:
+      description: "Als een verblijfsobject één van de gebruiksdoelen in deze lijst heeft, dan wordt het verblijfsobject in het zoekresultaat meegenomen."
+      name: gebruiksdoelen
+      in: query
+      required: false
+      schema:
+        type: array
+        minItems: 1
+        items:
+          $ref: '#/components/schemas/GebruiksdoelEnum'
+
+    oppervlakte:
+      name: oppervlakte
+      description: >-
+        Geef de minimale en maximale oppervlakte op waarbinnen je wilt zoeken (in m2). 
+        Bv: oppervlakte[min]=100&oppervlakte[max]=200
+      in: query
+      required: false
+      schema:
+        $ref: '#/components/schemas/IntegerFilter'
+      style: deepObject
+      explode: true
+
+    punt:
```

Comment:
Is het ten behoeve van consistentie niet beter om punt ook als een array te definieren? bbox en cirkel zijn nu gedefinieerd als arrays terwijl het ook als een object kan worden gedefinieerd.

Reply:
Punt is verwijderd.